src/style.css

```
@@ -0,0 +1,98 @@
+:root {
+  --darkColor: rgba(61, 78, 97, 0.1);
+  --lightColor: #3d4e61;
+  --borderColor: #e1e7ed;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+}
+
+.search {
+  margin: 20px 0;
```

you've got margin collapsing (big form margin-bottom and small form margin-top)
```suggestion
  margin: 20px 0 0;
```